main.tf

```diff
@@ -100,7 +113,8 @@ resource "aws_security_group" "ecs" {
   }
 
   tags {
-    Name = "ecs-sg-${var.name}"
+    Name      = "ecs-sg-${var.name}"
+    Terraform = "yes"
```

Comment:
I like these tags, one though,  what would you think about the tags reading "true" rather then "yes" ?

Reply:
I have zero preference. We use this pattern internally, but happy to follow whatever patterns the community/maintainers prefer.

Reply:
cool, lets switch this to `true` and I'll merge PR in.